### ISSUE TITLE
Option to LOCK the ringmenu so you don't accidentally move a icon.

### ISSUE THREAD

**Issue by nexitem** [jsb/RingMenu#25]
As Title suggests.

**Comment by jsb** [comment]
RingMenu's action buttons are built from the default Blizzard action buttons. So enabling the global "lock action bars" option in the interface settings should do the trick. Does that work for you?

**Comment by nexitem** [comment]
Yes this does work but I do like to have some bars locked and others not. because I sometimes change some skills around or put in new potions etc and I prefer not having to go into the menu :)

However I never accidentally move anything out of my action bars but with ring menu that seems to happen much more often ( especially now that I have changed the code to always open in the middle )

**Comment by jsb** [comment]
OK, I'll consider it as an option. Thanks for the suggestion.
